README.md

```diff
@@ -17,7 +17,14 @@ into your own Python scripts:
 
 ```
 import diyepw
-diyepw.create_amy_epw_files_for_years_and_wmos([723154, 772104], [2010, 2011, 2012])
+diyepw.create_amy_epw_files_for_years_and_wmos(
```

Comment:
need to switch up the years and the WMOs for this example code

Also, get some errors for year 2012 (possibly because leap year?): 

```
Creating AMY EPW for year 2012 and WMO 725300...
Problem processing year 2012 and WMO index 725300: 
                    This meteorological year has 8760 observations, but you passed a 
                    list of 8784 replacement values
```

Reply:
Oops, yeah, we don't support leap years yet. It's been on my TODO list for a couple of months. I'll fix that in this PR, it's getting too close to the end of my time to leave it any longer.

Reply:
Done.

Reply:
The README is fixed, too, thanks for spotting that!